mxcubeweb/core/components/user/usermanager.py

```diff
@@ -237,6 +237,7 @@ def sso_token_expired(self) -> bool:
                     "grant_type": "refresh_token",
                     "refresh_token": current_user.refresh_token,
                 },
+                timeout=5,
```

Comment:
It might be better to use a named constant

```suggestion
                timeout=HTTP_REQUESTS_TIMEOUT,
```

With something like this at the top:

```python
HTTP_REQUESTS_TIMEOUT = 5  #: Generic timeout for HTTP request in seconds
```